scripts/generate_sample_projects/main.py

```diff
@@ -0,0 +1,217 @@
+"""Generate sample projects based on the Zoo data on /ceph/zoo.
+
+This script can generate 4 sample projects for a given species/experiment.
+The generated projects will be saved in a local data repository - DATA_REPO_LOCAL,
+which should be a clone of https://gin.g-node.org/SainsburyWellcomeCentre/WAZP.
+
+DATA_REPO_LOCAL
+└── SPECIES
+    ├── short-clips_raw
+    ├── short-clips_compressed
+    ├── entire-video_raw
+    └── entire-video_compressed
+
+"short-clips_raw" contains ~10s clips extracted from the input videos.
+"short-clips_compressed" contains compressed versions of the clips in short-clips_raw.
+"entire-video_raw" holds an entire input video (use for critical tests only).
+"entire-video_compressed" contains a compressed version of the entire input video.
+
+Each of the above 4 sample projects has the following structure:
+PROJECT
+    └── videos
+        ├── list of video files *.avi or *.mp4
+        └── list of video metadata files *.metadata.yaml
+    └── pose_estimation_results
+        └── list of *.h5 files
+    ├── WAZP_config.yaml
+    └── metadata_fields.yaml
+
+The latter two files are created based on template files present in this directory.
+"""
+
+import shutil
+from pathlib import Path
+
+from file_utils import check_file_io_safety, update_wazp_config_file
+from pose_utils import extract_clip_from_poses
+from video_utils import compress_video, copy_entire_video, extract_clip_from_video
+
+###############################################################################
+# Modify these variables according to your system and needs
+###############################################################################
+
+# Path to the local data repository
+# This should be a clone of https://gin.g-node.org/SainsburyWellcomeCentre/WAZP
+DATA_REPO_LOCAL = Path("/home/nsirmpilatze/Code/Data/WAZP/")
```

Comment:
Well, if we are going to make it into a command-line script, I would have to pass multiple arguments, not just the `DATA_REPO_LOCAL`. And we would have to do some restructuring as well, especially because some of the logic depends on the data structure in `ceph/zoo`. 

So I will skip the cli for now.